typescript/best-practice-web/README.md

```diff
@@ -0,0 +1,61 @@
+# Best Practices for Web Testing
+
+This repository was carefully created by the Solutions Architects at Sauce Labs using their decades of testing experience.
+
+A good testing strategy doesn't only focus on web testing. Instead, it tackles risks at all levels of the system.
+
+In this repository, you will find a cohesive Greybox (we have some insight into the app code) testing strategy using:
+
+✅ Browser testing on desktop
+
+✅ Browser testing on mobile
+
+✅ Visual testing
+
+✅ Front-end performance testing
+
+✅ CICD pipeline executed on push and PR
+
+✅ Sauce Labs cloud
+
+## Test Strategy
+
+| Expected Behavior  | Tested? | Test Type  | Rationale  | Tech |
+|---|---|---|---|---|
+| Application renders  | ✅ | Functional UI | A functional browser test checks that the app is okay | WebdriverIO |
+| Every web page of the app renders successfully on desktop | ✅ | Visual test | A visual test efficiently validates app rendering | WebdriverIO, Screener.io |
+| Every web page of the app renders successfully on mobile  | ✅ | Visual test on real mobile devices | A visual test efficiently validates app rendering | WebdriverIO, Screener.io |
+| A user can successfully check out  | ✅ | Functional web test  | A web test is a good way to check functionality  | WebdriverIO |
+| Front-end performance is at least an A  | ✅ | Front-end performance test  | Front-end perf is an important aspect of any digital quality effort | WebdriverIO, Sauce Labs |
+| Test code runs on every commit in under 5 minutes  | ✅ | CICD  | Slow feedback makes it hard to iterate  | Github Actions |
+| App is secure  | 🙅‍♂️ | Not covered here, but something to consider for testing strategy  |   |
+| App is accessibility friendly  | 🙅‍♂️ | Not covered here, but something to consider for testing strategy |   |
+
+## Clone repo and run tests
+
+You can also simply 
+* clone this repository
+* `cd webdriver/web/examples/typescript`
```

Comment:
@nikolay-advolodkin 

What is the folder that you are going to work from? This file is added to `root(demo-js)/typescript/best-practice-web`, but you explanation is `cd webdriver/web/examples/typescript`
I think we should keep the same structure as with all other folders, where best-practices is part of or web, or appium-web or appium-app

Reply:
@wswebcreation i didn't update the README yet. I'd like `root(demo-js)/typescript/best-practice-web` as the pwd. Where would you like to see this?